src/erl.rs

```diff
@@ -767,6 +767,8 @@ fn var<'a>(name: &'a str, constructor: &'a ValueConstructor, env: &mut Env<'_>)
             _ => atom(record_name.to_snake_case()),
         },
 
+        ValueConstructorVariant::Inline => todo!(),
```

Comment:
I think we'd render the identity function here.

```suggestion
        } => "fun(A) -> A end".to_doc()
```